vmdb/db/migrate/20150401223517_container_nodes_hardware.rb

```diff
@@ -0,0 +1,16 @@
+class ContainerNodesHardware < ActiveRecord::Migration
```

Comment:
I'd prefer if this was the literal hardwares table, and not a brand new table.  I'm not sure why this table is needed (or when it was added for that matter.  Additionally, migrations should be named with an action word/phrase like `CreateContainerNodesHardware` or `UpdateContainerNodesHardwareWithIndentities`